lib/_http_server.js

```diff
@@ -481,7 +481,7 @@ function onParserExecuteCommon(server, socket, parser, state, ret, d) {
 
       // TODO(isaacs): Need a way to reset a stream to fresh state
       // IE, not flowing, and not explicitly paused.
-      socket._readableState.flowing = null;
+      socket._setFlowing(null);
       server.emit(eventName, req, socket, bodyHead);
```

Comment:
while we are at it, can we remove the `TODO(isaacs)` and call this `_resetFlowing()` instead?